wallet-core/src/ffi.rs

```diff
@@ -196,10 +263,215 @@ pub unsafe fn pick_notes(
 
 /// Gets the bookmark from the given note.
 #[no_mangle]
-pub unsafe fn bookmark(leaf_ptr: *const u8, bookmark: *mut u64) -> ErrorCode {
-    let leaf: NoteLeaf = mem::from_buffer(leaf_ptr)?;
+pub unsafe fn bookmarks(
+    notes_ptr: *const u8,
+    bookmarks_ptr: *mut *mut u8,
+) -> ErrorCode {
+    let notes: Vec<NoteLeaf> = mem::from_buffer(notes_ptr)?;
+
+    let bookmarks: Vec<u64> =
+        notes.into_iter().map(|leaf| *leaf.note.pos()).collect();
+
+    let bytes: Vec<u8> = bookmarks
+        .iter()
+        .flat_map(|&num| num.to_le_bytes())
```

Comment:
Would be helpful to document somewhere that the note positions are stored in memory as little endina